src/site/xdoc/newsarchive.xml

```diff
@@ -34,6 +34,15 @@
         </div>
       </div>
 
+   <subsection name='2016'>
+
+     <a name="20160208"><h4>Feb 2016 - James modules merge into james-project</h4></a>
+     <p>Some of James modules were merged into <a href="https://git-wip-us.apache.org/repos/asf?p=james-project.git;a=tree">james-project</a>.</p>
+     <p>This change was done in order to fasten our development phase.</p>
```

Comment:
you could end the sentence with something like : "before the 3.0 release and as long as important changes are needed on API"